save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt

```diff
@@ -83,14 +97,17 @@ class SandboxController(
     fun uploadFile(
```

Comment:
there is `uploadFileAsText`, probably it should be changed too

Reply:
Oh sure